### PR TITLE
PHP components of the code style guide

### PR DESCRIPTION
**Pull request by dealancer**
See https://trello.com/c/IrJpldKi/104-php-components-of-the-code-style-guide-vad-m.

### CODE REVIEW

File: codeguide.md [hunk 1]
```diff
@@ -80,7 +80,29 @@ The code style for Twig files mostly follows the above standards for HTML, with
 
 ## PHP
 
-Our PHP style guide very closely follow the [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/).
+Our PHP style guide very closely follow the [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) with a few exceptions (see below).
```
Comment (thatdevgirl) [inline codeguide.md]: Super nit-picky grammar thing: "follow" should be "follows".

File: codeguide.md [hunk 2]
```diff
@@ -80,7 +80,29 @@ The code style for Twig files mostly follows the above standards for HTML, with
...
+
+### Tab indents
+
+- Use space tabs instead of real tabs, so it is concistent across all of our code.
```
Comment (thatdevgirl) [inline codeguide.md]: Nit-picky spelling thing: "consistent"

File: codeguide.md [hunk 3]
```diff
@@ -80,7 +80,29 @@ The code style for Twig files mostly follows the above standards for HTML, with
...
+- Use two spaces for one indent.
+
+### Class naming
```
Comment (thatdevgirl) [inline codeguide.md]: I really like this part - it follows the naming conventions for Javascript between classes and variables.

File: codeguide.md [hunk 4]
```diff
@@ -80,7 +80,29 @@ The code style for Twig files mostly follows the above standards for HTML, with
...
+
+- Use capitalized words for the class names. Do not use underscore for sepration between words:
```
Comment (thatdevgirl) [inline codeguide.md]: Nit-picky spelling thing: "separation"

File: codeguide.md [hunk 5]
```diff
@@ -80,7 +80,29 @@ The code style for Twig files mostly follows the above standards for HTML, with
...
+````
+class MyClass; // Correct
+class myclass; // Incorrect
+class My_Class; // Incorrect
+````
+- When creating instance of the calss make sure first letter is lover case:
```
Comment (thatdevgirl) [inline codeguide.md]: Nit-picky spelling thing: "class" and "lower"

File: codeguide.md (hunk 1 above)
Comment (thatdevgirl) [reply codeguide.md]: Also, I think we should take a stance on using quotes. The PHP style guide is basically saying that it is OK to use either. 

The rest of our style guide sets single quotes for string literals and values. (At least this is true for Twig, JS, and Sass. HTML is an exception.

**Comment by dealancer** [comment]
`follow` was not mine but yes it was a bad grammar day. Shakespeare might be crying on this.

**Comment by dealancer** [comment]
@thatdevgirl i like the way how PHP style guide is saying about quotes, e.g. use single, unless double is needed, but if you have better ideas, feel free to express them.

**Comment by dealancer** [comment]
@thatdevgirl fixed typos and add *quotes* section.

**Comment by dealancer** [comment]
@thatdevgirl thanks, now you know how I write, when auto spellchecking is off.

**Comment by dealancer** [comment]
yay!

**Comment by dealancer** [comment]
@reedcodes do you approve PR?

**Pull request merged**